examples/docker/setup.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+# fail on first error
+set -e
+
+#
+# Work in temp directory (see Dockerfile)
+#
+WORKING_DIR=$(pwd)
+
+#
+# Install boost 1.44
+#
+cd $WORKING_DIR
+if ! [ -f "boost_1_44_0.tar.bz2" ]; then
+	wget -nv 'http://downloads.sourceforge.net/project/boost/boost/1.44.0/boost_1_44_0.tar.bz2'
+fi
+tar xjf boost_1_44_0.tar.bz2
+cd boost_1_44_0
+./bootstrap.sh
+./bjam --build-type=complete --layout=versioned --with-system --with-thread threading=multi link=shared install
+
+#
+# Update ldconfig cache
+#
+echo '/usr/local/lib' > /etc/ld.so.conf.d/user-local-lib.conf
+ldconfig -v
+
+#
+# Install LuaRocks
+#
+cd $WORKING_DIR
+if ! [ -f "luarocks-2.1.1.tar.gz" ]; then
+	wget -nv http://luarocks.org/releases/luarocks-2.1.1.tar.gz
+fi
+tar xzf luarocks-2.1.1.tar.gz
+cd luarocks-2.1.1
+./configure
+make build
+make install
+
+#
+# Install LuaNode
+#
+cd $WORKING_DIR
+mkdir luanode
+cd luanode
+LUANODE_BASE_DIR=$(pwd)
+mkdir -p sources
+mkdir -p bin
+cd sources
+git clone git://github.com/ignacio/LuaNode.git
+cd LuaNode/build/linux
```

Comment:
It would be less painfull to use CMake here. It has no INCONCERT_DEVEL nonsense :wink: